src/script/addRemove.js

```diff
@@ -149,7 +150,6 @@ document.querySelector('.list').addEventListener('change', (e) => {
   if (e.target.classList.contains('taskStatus')) {
     console.log(e.target.nextElementSibling);
```

Comment:
- [ ] For javascript best practices I think it would be nice if you remove all `console.log` in your project.

Reply:
Asked change implemented